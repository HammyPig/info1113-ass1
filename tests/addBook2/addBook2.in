add book books.csv 111111
aDD BOOK books.csv 111112
ADD BOOK books.csv 111111
LIST ALL