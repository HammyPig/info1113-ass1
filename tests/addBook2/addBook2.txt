cannot add books with the same serial number