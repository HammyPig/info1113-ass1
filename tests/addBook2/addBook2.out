user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Successfully added: The Hobbit (J.R.R. Tolkien).

user: Book already exists in system.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)

user: Ending Library process.
